deployments/deploy.js

```diff
@@ -4,11 +4,12 @@ async function main() {
     const UsersInfo = await ethers.getContractFactory("UsersInfo");
     const user_info = await UsersInfo.deploy();
     console.log("Contract Deployed to Address:", user_info.address);
+    console.log("Nice")
```

Comment:
This console log statement is unnecessary and can be removed to keep the codebase clean and avoid cluttering the console output during deployment.